weave/trace_server_bindings/remote_http_trace_server.py

```diff
@@ -161,6 +161,41 @@ def _flush_calls(
             self._flush_calls(batch[split_idx:], _should_update_batch_size=False)
             return
 
+        final_batch = []
+        if encoded_bytes > CLICKHOUSE_SINGLE_ROW_INSERT_BYTES_LIMIT:
```

Comment:
it would be nice to somehow mark the call or its inputs / outputs in these cases so we know that this happened and can surface that info to the user

Reply:
Yeah, I think the right thing is to go with the JSONBlob serializer fix instead. This is more of a workaround...